tests/basic_event_manager.rs

```diff
@@ -93,15 +93,15 @@ fn test_single_threaded() {
     app.run();
 
     let counters = app.get_counters();
-    for i in 0..100 {
-        assert_eq!(counters[i], 1 & (triggered_subscribers.contains(&i) as u64));
+    for (i, &item) in counters.iter().enumerate().take(100) {
```

Comment:
We shouldn't use take here anymore. We can just enumerate all counters.

Reply:
Removed `take`.